transmart-oauth/grails-app/init/org/transmart/oauth/Application.groovy

```diff
@@ -0,0 +1,16 @@
+package org.transmart.oauth
+
+import grails.boot.*
+import grails.boot.config.GrailsAutoConfiguration
+import grails.plugins.metadata.*
+import groovy.util.logging.Slf4j
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.autoconfigure.security.SecurityFilterAutoConfiguration
+
+@Slf4j
+@EnableAutoConfiguration(exclude = [SecurityFilterAutoConfiguration])
```

Comment:
The other plugins have `@PluginSource` here instead of these annotations. Is this correct?